docs/js-guide/static-files.rst

```diff
@@ -1,19 +1,104 @@
-Static Files
-============
+Overview of Static Files and JavaScript Bundlers
+================================================
+
+What are Static Files?
+----------------------
 
 Static files include any ``css``, ``js``, and image files that are not
 dynamically generated during runtime. ``css`` is typically compiled from
-``less`` and minified prior to server runtime. ``js`` files are
-collected, combined, and minified prior to server runtime. As of this
-writing we don’t compile our JavaScript from a higher level scripting
-language. Image files generally stay as-is. The only ‘dynamic’ images
+``scss`` (or ``less`` on Bootstrap 3 pages) and minified before server
+runtime.
+
+``js`` files are collected, combined, and minified using Webpack
+or RequireJS, called JavaScript bundlers. We are currently
+transitioning from RequireJS (deprecated) to Webpack. Some pages do
+not use a bundler or any structured JavaScript module format,
+referred to as No-Bundler Pages. No-Bundler Pages are rare and are being transitioned
+to using Webpack as part of the `JS Bundler Migration
+<https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/migrating.rst>`__.`
+
+Image files generally stay as-is. The only "dynamic" images
 come from file attachments in our database.
 
-Due to their static natures, the primary objective for static files is
-to make as few requests to them during page load, meaning that our goal
-is to combine static files into one larger, minified file when possible.
-An additional goal is to ensure that the browser caches the static files
-it is served to make subsequent page loads faster.
+As of this writing, we don't compile our JavaScript from a higher level scripting
+language, although Webpack offers ways to do this easily.
+
+Due to their static natures, the primary objective when working with static files is
+to make as few requests as possible to them during page load. We aim to combine
+static files into one larger, minified file whenever possible.
+Another goal is to ensure that the browser caches the static files when possible
+to make subsequent page loads faster.
+
+
+Why use a javascript bundler?
+-----------------------------
+
+We use a bundler for our javascript files to reduce the amount of separate
```

Comment:
Nitpick: "script tags" => "network requests"

Reply:
eh, actually I don't think it matters

Reply:
ended up changing it anyway... script tags was the original verbiage i just moved around